infrastructure/src/main/java/org/corfudb/infrastructure/management/ClusterStateAggregator.java

```diff
@@ -6,11 +6,15 @@
 import lombok.NonNull;
 import org.corfudb.protocols.wireprotocol.ClusterState;
 import org.corfudb.protocols.wireprotocol.NodeState;
+import org.corfudb.protocols.wireprotocol.SequencerMetrics;
+import org.corfudb.protocols.wireprotocol.failuredetector.NodeConnectivity;
 import org.corfudb.protocols.wireprotocol.failuredetector.NodeConnectivity.NodeConnectivityType;
 
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
```

Comment:
It looks like the base case when `clusterStates.size() = 1` is already covered by the `reduce()` method.

Reply:
Yes.